tools/retest/main.go

```diff
@@ -93,7 +94,11 @@ issues:
 			continue
 		}
 		log.Printf("#%d jobs to retest: %s", prNumber, strings.Join(jobsToRetest, ", "))
-		newComments := commentsToCreate(statuses, jobsToRetest, shouldRetestFailedStatuses(statuses, userComments))
+		shouldRetestErr := shouldRetestFailedStatuses(statuses, userComments)
+		if shouldRetestErr != nil {
+			log.Printf("PR %d retest status due to failures: %v", prNumber, shouldRetestErr)
```

Comment:
Let's try to keep a consistent log style where easily possible:
```suggestion
			log.Printf("#%d not issuing /retest: %v", prNumber, shouldRetestErr)
```

Reply:
This is that special case where the text could be confusing, as there are also other places where we decide whether to retest or not.